syfertext/utils.py

```diff
@@ -52,36 +51,29 @@ def get_lang_model(model_name: str):
 
         os.mkdir(data_path)
 
-        # a flag signified that the model show be downloaded
-        download_model = True
-
     # If the data folder does not contain the language model
     # folder, download the language model
     if model_name not in os.listdir(data_path):
 
         download_model = True
 
-        # full path of the model folder to create
-        model_path = os.path.join(data_path, model_name)
-
-        # Create the model folder
-        os.mkdir(model_path)
-
     if download_model:
 
-        # download model files into the specified path
-        _download_model(model_name, model_path)
+        # download model files into a temporary path
+        tmp_path = _download_model(model_name)
+
```

Comment:
please call tmp_path -> tmp_model_path

To remove ambiguity with the name _tmp_path you used in _download_model()